components/calendar/src/coptic.rs

```diff
@@ -15,7 +15,8 @@ use core::marker::PhantomData;
 use tinystr::tinystr;
 
 #[derive(Copy, Clone, Debug, Hash, Default, Eq, PartialEq)]
-// The Coptic Calendar
+#[allow(clippy::exhaustive_structs)] // this type is stable
```

Comment:
Question: Is it true that this type is stable? The only reason this shows up in the lint is because it is an empty struct. Do we really want to guarantee that the struct will always be empty? (also elsewhere in this crate)

Reply:
In this _specific_ case we're actually going to be adding a boolean to it, so no, but I don't think the others should ever get a field.